continuous_integration/install_coverage_subprocess_pth.py

```diff
@@ -5,8 +5,10 @@
 import os.path as op
 from distutils.sysconfig import get_python_lib
 
-FILE_CONTENT = u"""\
-import coverage; coverage.process_startup()
+FILE_CONTENT = u"""
+import os, coverage
+if \"COVERAGE_PROCESS_START\" in os.environ:
+    coverage.process_startup()
```

Comment:
Also "coverage.process_startup()" does nothing if the `COVERAGE_PROCESS_START` variable is not set.